.github/workflows/tests.yml

```
@@ -29,7 +25,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: ddev/github-action-add-on-test@v1
+      - uses: ddev/github-action-add-on-test@v2.3.1
```

I think this would be better.

```suggestion
      - uses: ddev/github-action-add-on-test@v2
```

---

True, i'd love to have this but it still serves the old version containing the keepalive workflow

---

I'm seeing that also in https://github.com/ddev/ddev-platformsh/actions/runs/14600768122 but I don't understand it. Of course using 2.3.1 will mean manual updates forever.

---

🤔 I'll set it to @v2 hoping this issue will just disappear tomorrow.

Thanks.

---

Both @stasadev and I can only guess that somehow github caches that.

---

Looks to me like I did something wrong with the release. It's supposed to automatically push the major version.

---

It should work now. @stasadev moved the tag.

---

Yes, thanks a bunch. It's working now.